scenarios/kubernetes_kubelet.py

```diff
@@ -53,14 +53,20 @@ def var(path):
     """Expands '${foo} interesting' to 'something interesting'."""
     return os.path.expandvars(path)
 
+def get_tag(branch):
+    #If branch 3-part version, only take the first two parts
+    release = re.match(r'release-(\d+\.\d+)', branch)
```

Comment:
For some reason, in this file the `VERSION_TAG` keys include the `release-` prefix. Not sure if anything else is depending on that...

Reply:
hummm, good catch, yeah the image tag is different

Reply:
@kow3ns to match previous behavior I think you want to use `match.group(0)` here instead

Reply:
@krzyzacy 
Match group(1) contains the version (e.g. 1.2 or 1.3). BRANCH_VERSION gets version associated with the branch and VERSION_TAG gets the tag associated with that version.

Reply:
okay, then you'll need to fix `VERSION_TAG` above as well (trim `release-` part)

Reply:
I think for `release-1.7`, `VERSION_TAG[BRANCH_VERSION.get('1.7', '1.7')]` will not have an entry now?